www/src/content/docs/docs/continuous-integration.mdx

```diff
@@ -0,0 +1,127 @@
+---
+title: Continuous Integration
+description: Configure GitHub Actions to deploy your application.
+---
+
+To deploy your application from GitHub Actions, you need to setup two things:
+
+1. An OIDC Provider and IAM Role
+2. A GitHub Action workflow
+
+---
+
+## OIDC Provider and Role
+
+In your `sst.config.ts`, add the following code:
+
+```ts
+const isPersonalStage = $app.stage !== "production" && $app.stage !== "dev";
+if (!isPersonalStage) {
+  const oidc = new aws.iam.OpenIdConnectProvider("GithubOidc", {
```

Comment:
If you're using the same AWS account for prod and dev, that's where you'll have issues. The providers aren't namespaced and exist at a global level in the AWS account. I only deploy this to production as I don't want my github action deploying to a dev stage, which is why I haven't run into this issue.

Reply:
@mattkinnersley Yup figured that out, forgot to mention.
I'll still suggest this to be:
```ts
const oidc = new aws.iam.OpenIdConnectProvider(`GithubOidc_${$app.stage}`,{})
```
Since the documentation will be read by users who'll be most probably starting up with single AWS account and they may encounter this error

Reply:
SST is usually pretty good at prefixing components with the stage elsewhere- I guess that's not the case here?